com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/validators/parameters/RestartOnEnvChangeValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+package com.sap.cloud.lm.sl.cf.core.validators.parameters;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import com.sap.cloud.lm.sl.mta.model.v1.Module;
+
+import jersey.repackaged.com.google.common.collect.ImmutableMap;
```

Comment:
Do not use this package:
https://stackoverflow.com/questions/34930415/why-jersey-repackaged-guava

Reply:
Any other suggestions for creating and initialize map at one line? If there is not other solution I will have to add new dependency for Apage Guava.